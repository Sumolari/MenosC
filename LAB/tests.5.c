int valint = 5;
int intval = 5;
