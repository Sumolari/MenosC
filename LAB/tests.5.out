  1- Tipo entero: int
  1- Identificador: valint
  1- Operador ASIGNACIÓN: =
  1- Constante Entera: 5
  2- Nueva Linea
  2- Tipo entero: int
  2- Identificador: intval
  2- Operador ASIGNACIÓN: =
  2- Constante Entera: 5
  3- Nueva Linea
