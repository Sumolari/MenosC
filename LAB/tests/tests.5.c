{
	int valint;
	valint = true;
	 int intval;
	          intval = true || 5 && 90 + 1;
}