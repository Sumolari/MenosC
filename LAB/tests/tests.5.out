Linea 3: Tipo incompatible
Linea 5: Tipo incompatible

Numero de errores:      2
